deployer/src/persistence/resource/mod.rs

```diff
@@ -81,6 +81,7 @@ pub enum Type {
     Persist,
     Turso,
     Metadata,
+    Opendal,
```

Comment:
I'm not sure if `Opendal` is a good name, maybe we can use `Blob` or `Storage` to better describe what it does?

Reply:
It would be better to not add a variant here and instead use `Custom`. The reason for this is that we are trying to use this enum for resources that Shuttle can manage, and all client plugins are separate.